src/OpenLoco/src/Objects/ObjectIndex.cpp

```diff
@@ -56,17 +56,17 @@ namespace OpenLoco::ObjectManager
     static loco_global<ObjectSelectionMeta, 0x0112C1C5> _objectSelectionMeta;
     static loco_global<std::array<uint16_t, kMaxObjectTypes>, 0x0112C181> _numObjectsPerType;
 
-    static constexpr uint8_t kCurrentIndexVersion = 4;
+    static constexpr uint8_t kCurrentIndexVersion = 5;
+    static constexpr uint32_t kMaxStringLength = 1024;
 
     struct ObjectFolderState
     {
         uint32_t numObjects = 0;
         uint32_t totalFileSize = 0;
         uint32_t dateHash = 0;
-        constexpr bool operator==(const ObjectFolderState& rhs) const
-        {
-            return (numObjects == rhs.numObjects) && (totalFileSize == rhs.totalFileSize) && (dateHash == rhs.dateHash);
-        }
+        std::string basePath = "";
+
+        constexpr auto operator<=>(const ObjectFolderState& rhs) const = default;
```

Comment:
Seems our Ubuntu jammy CI doesn't like this being declared constexpr.
```suggestion
        auto operator<=>(const ObjectFolderState& rhs) const = default;
```